components/_datasets/Chicago Taxi/components.yaml

```diff
@@ -0,0 +1,33 @@
+name: Chicago Taxi Trips dataset
```

Comment:
I think it worth a .md doc to briefly introduce how to use this. Especially how to map each parameter to corresponding API query syntax. WDYT?

Reply:
I'll do this.
I'll also create a sample that uses this component.

Reply:
For now I've added extended information about the component and query syntax to the description of the component.